client/src/components/Navbar.js

```diff
@@ -1,116 +1,107 @@
+  if (auth?.user) {
+    loginRedirect = authLevelRedirect(auth.user);
+  }
 
-    // check user accessLevel and adjust link accordingly
-    const [page, setPage] = useState('home');
-    const { auth } = useAuth();
-    let loginRedirect = '/admin';
-    if (auth?.user) {
-      loginRedirect = authLevelRedirect(auth.user);
-    }
-    const notAuth = () => (
-        <>
-          {page === 'home' ?
-            <p className="nav-link-text nav-link-active">CHECK-IN</p>
-            :
-            <div className='nav-link-container'>
-              <Link to="/" onClick={e => setPage('home')} >
-                <p className="nav-link-text">
-                  CHECK-IN
-                </p>
-              </Link>
-            </div>}
-          {page === 'adminLogin' ?
-            <p className="nav-link-text nav-link-active">ADMIN</p>
-            :
-            <div className='nav-link-container'>
-              <Link to={loginRedirect} onClick={e => setPage('adminLogin')}>
-                <p className="nav-link-text">
-                  ADMIN
-                </p>
-              </Link>
-            </div>
-            }
-        </>
-    )
-    const isAuth = () => (
-      <>
-        {page === 'useradmin' ?
-          <p className="nav-link-text nav-link-active" >USER MANAGEMENT</p>
-          :
-          <div className='nav-link-container'>
-            <Link to="/useradmin" onClick={e => setPage('usermanagement')}>
-              <p className="nav-link-text">ADMIN</p>
-            </Link>
-          </div>}
-        {page === 'projects' ?
-          <p className="nav-link-text nav-link-active" >PROJECTS</p>
-          :
-          <div className='nav-link-container'>
-            <Link to="/projects" onClick={e => setPage('projects')}>
-              <p className="nav-link-text">PROJECTS</p>
-            </Link>
-          </div>}
-      </>
-    )
-    const isUser = () => (
-      <>
-        {page === 'home' ?
-          <p className="nav-link-text nav-link-active" >PROJECTS</p>
-          :
-          <div className='nav-link-container'>
-            <Link to="/projects" onClick={e => setPage('home')}>
-              <p className="nav-link-text">PROJECTS</p>
-            </Link>
-          </div>}
-      </>
-    )
-
-
-    return (
-      <div className="nav-wrapper">
-        <nav className="navbar" role="navigation" aria-label="main navigation">
-          <div className="navbar-buttons-container">
-
-            {!auth?.user &&
-              notAuth()
-            }
-
-            {auth?.user?.accessLevel === 'admin' &&
-              isAuth()
-            }
+  const [selected, setSelected] = useState(1);
 
-            {auth?.user?.accessLevel === 'user' &&
-              isUser()
-            }
+  // Styles that may need to be included in the theme.
+  const StyledButton = styled(Button)({
+    color: 'black',
+    marginLeft: '2rem',
+    padding: '0.1rem 0.5rem',
+    borderRadius: 0,
+    fontSize: '1rem',
+  });
```

Comment:
If you add `exact` to your `NavLink` components, you might not have to use state

See docs for clarification: https://v5.reactrouter.com/web/api/NavLink/exact-bool

and then assuming the `.active` class is being applied and removed properly by react-router-dom,
I think you might be able to just add 

````
"& .active": {
      borderBottom: "2px #fa114f solid"
}

```

Reply:
Taken care of in next commit.